### DISCUSSION TITLE
renaming getter function in createSignal

### DISCUSSION THREAD

**Discussion by Ashutosh94g** [solidjs/solid#987] [Ideas]
As we know that when we create a signal in Solidjs we get a getter and a setter function but the naming convention of the getter is value in the docs. for example
const [**state**, setState] = createSignal(false);
here due to some react vibe we name getter as a value which is accurate for react js but in our case, it should be more like
const [**getState**, setState] = createSignal(false);
This will directly indicate two things that are a getter function, not a value, and avoid a lot of confusion

**Comment by Exelord** [comment]
Fully support that! That naming caused many misunderstanding in for new adopters.

**Comment by petercockerell-brt** [comment]
I think naming is entirely down to what makes you comfortable. I've just stored the result from `createSignal()` directly (un-destructured) on occasion, and accessed the getter setter via `s[0]` and `s[1]`. I don't think Solid cares! I think the only downside to your proposal is more typing 😀
